subcommands/keys/tuf_rotate_offline_key.go

```diff
@@ -97,22 +99,32 @@ Instead, please, use the "fioctl keys tuf rotate-offline-key --role=targets --cr
 	tempSyncProdRoot.Flags().StringP("creds", "o", "", "Offline creds file containing root keys.")
 	_ = tempSyncProdRoot.MarkFlagRequired("creds")
 	_ = tempSyncProdRoot.MarkFlagFilename("creds")
+	tempSyncProdRoot.Flags().StringP("targets-creds", "O", "", "Offline creds file containing targets keys.")
+	_ = tempSyncProdRoot.MarkFlagFilename("creds")
 	tufCmd.AddCommand(tempSyncProdRoot)
 }
 
 func doSyncProdRoot(cmd *cobra.Command, args []string) {
 	factory := viper.GetString("factory")
 	credsFile, _ := cmd.Flags().GetString("creds")
+	targetsCredsFile, _ := cmd.Flags().GetString("targets-creds")
 	root, err := api.TufRootGet(factory)
 	subcommands.DieNotNil(err)
 	creds, err := GetOfflineCreds(credsFile)
 	subcommands.DieNotNil(err)
-	subcommands.DieNotNil(syncProdRoot(factory, root, creds))
+	if targetsCredsFile == "" {
+		subcommands.DieNotNil(syncProdRoot(factory, root, creds, creds))
```

Comment:
shouldn't this be `subcommands.DieNotNil(syncProdRoot(factory, root, creds, nil))`?

Reply:
It should be like present in the code.
The user might choose to keep both root and targets keys in one tarball; actually in the past it was the only way to do this.
Thus, we pass `creds, creds` so that the code have where to search for targets keys if needed (to re-sign prod targets).